kpi/views/v2/asset.py

```diff
@@ -238,12 +246,37 @@ def get_serializer_context(self):
             ).order_by(
                 'user__username', 'permission__codename'
             )
-            object_permissions_per_object = defaultdict(list)
+            object_permissions_per_asset = defaultdict(list)
 
             for op in object_permissions:
-                object_permissions_per_object[op.object_id].append(op)
+                object_permissions_per_asset[op.object_id].append(op)
+
+            context_['object_permissions_per_asset'] = object_permissions_per_asset
+
+            # 3) Get the collection subscriptions per asset
+            subscriptions_queryset = UserAssetSubscription.objects. \
+                values('asset_id').annotate(user_subscriptions=Count('user_id'))
+
+            user_subscriptions_per_asset = defaultdict(int)
+            for record in subscriptions_queryset:
+                user_subscriptions_per_asset[record['asset_id']] = \
+                    record['user_subscriptions']
+
+            context_['user_subscriptions_per_asset'] = user_subscriptions_per_asset
+
+            # 4) Get children count per asset
+            # ToDo Verify if all children must be included in count (e.g. Archives)
```

Comment:
not sure if i commented on this before (i know the same question appears in a different place in this PR), but only deployed assets can be archived, and none of those should be inside a collection (library assets cannot be deployed)